Prometheus/Summary.cs

```diff
@@ -152,11 +141,21 @@ private protected override async Task CollectAndSerializeImplAsync(IMetricsSeria
                     }
                 }
 
-                await serializer.WriteMetricAsync(_sumIdentifier, sum, cancel);
-                await serializer.WriteMetricAsync(_countIdentifier, count, cancel);
+                await serializer.WriteIdentifierPartAsync(Parent.NameBytes, FlattenedLabelsBytes, cancel,
```

Comment:
Would it make sense to still keep it 1 call on this level and split into identifier/value parts inside the TextSerializer, just to keep the calling code simpler? I suppose this depends on what you have in mind for the next iteration. Ideally we can put all the complexity in the serializer but it could be I am thinking too simplistically here.

Reply:
Thank you for the push. I have added a `WriteMetricPointAsync` function and encapsulated the others away. OpenMetrics encoding concerns should fit that single function 🤞 .